governance/cla/index.html

```diff
@@ -1,3 +1,202 @@
+        <link rel="apple-touch-icon" sizes="60x60" href="assets/favicons/apple-touch-icon-60x60.png">
+        <link rel="apple-touch-icon" sizes="72x72" href="assets/favicons/apple-touch-icon-72x72.png">
+        <link rel="apple-touch-icon" sizes="76x76" href="assets/favicons/apple-touch-icon-76x76.png">
+        <link rel="apple-touch-icon" sizes="114x114" href="assets/favicons/apple-touch-icon-114x114.png">
+        <link rel="apple-touch-icon" sizes="120x120" href="assets/favicons/apple-touch-icon-120x120.png">
+        <link rel="apple-touch-icon" sizes="144x144" href="assets/favicons/apple-touch-icon-144x144.png">
+        <link rel="apple-touch-icon" sizes="152x152" href="assets/favicons/apple-touch-icon-152x152.png">
+        <link rel="apple-touch-icon" sizes="180x180" href="assets/favicons/apple-touch-icon-180x180.png">
+        <link rel="icon" type="image/svg+xml" href="assets/favicons/favicon.svg">
+        <link rel="icon" type="image/png" sizes="32x32" href="assets/favicons/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="192x192" href="assets/favicons/android-chrome-192x192.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="assets/favicons/favicon-16x16.png">
+        <link rel="shortcut icon" href="assets/favicons/favicon.ico">
+        <link rel="manifest" href="assets/favicons/manifest.json">
+
+        <link rel="stylesheet" type="text/css" media="screen" href="../assets/css/all.css" />
+        <link rel="stylesheet" type="text/css" media="print" href="../assets/css/print.css">
+    </head>
+    <body>
+
+    <div class="global-wrapper">
+      <main class="main">
+
+        <div class="wrapper wrapper--content">
+          <div class="grid">
+            <div class="content-container grid__item one-whole large--ten-twelfths push--large--one-twelfth x-large--eight-twelfths push--x-large--two-twelfths grid-column">
+              <section class="text-center">
+                <h2 id="tmp-title">Fiduciary License Agreement 2.0</h2>
+                <span id="tmp-subtitle-based">based on the</span>
+                <h2 id="tmp-subtitle"><span id="tmp-contributor-type">Individual</span> Contributor <span id="tmp-contributor-exclusivity-1">exclusive</span> License Agreement</h2>
+                <h2 id="tmp-subtitle-patent">(including the <span id="tmp-patent-option">Traditional Patent License</span> OPTION)</h2>
+                </section>
+
+                <p>Thank you for your interest in contributing to <span id="tmp-beneficiary-name">eLife Sciences Publications, Ltd</span>'s <span id="tmp-project-name">Libero</span> ("We" or "Us").</p>
+
+                <p>The purpose of this contributor agreement ("Agreement") is to clarify and document the rights granted by contributors to Us. To make this document effective, please follow the instructions at <span id="tmp-submission-instructions">http://libero.pub/signup</span>.</p>
```

Comment:
Can you swap out the `<span>` with an `<a>` for the link to the signup page, please? Worth the effort even if we have to regenerate. NB: that page will exist once the community have approved.

Reply:
Replaced for link